engines/lab/processroom.cpp

```diff
@@ -326,7 +326,7 @@ void LabEngine::doActions(const ActionList &actionList) {
 				// This is a Wyrmkeep Windows trial version, thus stop at this
 				// point, since we can't check for game payment status
 				_graphics->readPict(getPictName(true));
-				GUI::MessageDialog trialMessage("This is the end of the trial version. You can play the full game using the original interpreter from Wyrmkeep");
+				GUI::MessageDialog trialMessage(Common::convertToU32String("This is the end of the trial version. You can play the full game using the original interpreter from Wyrmkeep"));
```

Comment:
Actually, this is a mistake, and it must be wrapped in `_()`, e.g. marked for translation.